src/spoof.ts

```diff
@@ -97,22 +99,37 @@ export const createCursor = (page: Page, start: Vector = origin): unknown => {
       }
       await page.mouse.up()
     },
-    async move (selector: string, options?: MoveOptions) {
-      if (options?.waitForSelector !== undefined) {
-        await page.waitForSelector(selector, {
-          timeout: options.waitForSelector
-        })
-      }
-
-      const elem = await page.$(selector)
-      if (elem === null) {
-        throw new Error(
-          `Could not find element with selector "${selector}", make sure you're waiting for the elements with "puppeteer.waitForSelector"`
-        )
+    async move (selector: string | ElementHandle, options?: MoveOptions) {
+      let elem
+      if (!isElementHandle(selector)) {
+        if (selector.includes('//')) {
+          if (options?.waitForSelector !== undefined) {
```

Comment:
I feel this would be a little bit nicer if you move the `waitForSelector` one level up - then this check is only once in the code.